File: sub-model/src/main/java/org/github/base/mapper/ISysUserRoleMapper.java
package org.github.base.mapper;

import org.github.base.entity.SysUserRoleEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * <p>
 * 用户和角色关联表 Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-20
 */
@MyBatisMapper
public interface ISysUserRoleMapper extends IMapper<SysUserRoleEntity> {}

File: sub-mysql-mydb-service/src/main/java/org/github/mysql/mydb/base/mapper/ISysUserRoleMapper.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.mydb.base.mapper;

import org.github.mysql.mydb.base.entity.SysUserRoleEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * 用户-角色关系表 Mapper 接口
 *
 * @author JYD_XL
 */
@MyBatisMapper
interface ISysUserRoleMapper : IMapper<SysUserRoleEntity>
